app/assets/stylesheets/common/base/topic.scss

```diff
@@ -80,6 +87,7 @@
 
 #topic-progress-wrapper {
   display: flex;
+  right: 9px;
```

Comment:
Curious because `9px` is very specific, does this match something somewhere else?